topiary-tutorial-part-1/tree-sitter-yolo/bindings/c/tree-sitter-yolo.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+
+Name: tree-sitter-yolo
+Description: @PROJECT_DESCRIPTION@
+URL: @PROJECT_HOMEPAGE_URL@
```

Comment:
I assume this just gets replaced with an empty string and that's fine. (Also, see L64 in your `Makefile`.)

Reply:
I guess? At any rate we just need the makefile to work (which I tested), I suspect we could even get rid of those packageconfig file. I assume it's not very important either way